server/src/testFixtures/java/org/elasticsearch/test/IntegTestCase.java

```diff
@@ -272,6 +274,7 @@
 @UseJdbc
 @UseHashJoins
 @UseRandomizedSchema
+@UseRandomizedOptimizerRules(alwaysKeep = {RemoveRedundantFetchOrEval.class, MergeFilterAndCollect.class})
```

Comment:
RemoveRedundantFetchOrEval and MergeFilterAndCollect have to be always activated otherwise a lot of basic select fail. This needs to be further investigated.